test_inventories.py

```diff
@@ -1,13 +1,23 @@
 import unittest
+from unittest.mock import MagicMock
+from inventory import Fetcher
 from inventory import InventoryBuilder
+import test_mocks as mocks
 import test_fixtures as fixtures
 
-
 class TestInventoryBuilderMethods(unittest.TestCase):
 
     def setUp(self):
+        self.inventory_fetcher = Fetcher(None, None)
```

Comment:
just use `self.fetcher`, got you're idea but no need to add the namespace in the prefix